pom.xml

```diff
@@ -133,10 +133,89 @@
 			<version>2.0.8</version>
 			<scope>test</scope>
 		</dependency>
-
 	</dependencies>
-	<build>
 
+    <profiles>
+        <profile>
+            <id>docker</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <!-- Copy the SQL scripts and modify them so they can be run from mysql -->
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <version>1.8</version>
+                        <executions>
+                            <execution>
+                                <id>copy-db-scripts</id>
+                                <phase>package</phase>
+                                <configuration>
+                                    <target>
+                                        <echo>ANT TASK - Copying database files and modifying for use with mysql</echo>
+                                        <copy todir="${basedir}/docker/mysql" overwrite="true" flatten="true">
+                                            <fileset dir="${basedir}/src/main/resources/database" includes="*.sql" >
+                                                <include name="*.sql" />
+                                            </fileset>
+                                        </copy>
+                                        <copy todir="${basedir}/docker/mongo" overwrite="true" flatten="true">
+                                            <fileset dir="${basedir}/src/database" includes="*.js" >
+                                                <include name="*.js" />
+                                            </fileset>
+                                        </copy>
+                                        <exec executable="sed">
```

Comment:
This fails on windows because there is no available `sed` command. Is there an alternative way to do this through maven? Or is this something that we'll have to specify to be used on Linux?

Reply:
There is a workaround in Windows for this. Need to document the pre-reqs